.github/workflows/e2e.yml

```diff
@@ -5,6 +5,9 @@ on:
     branches:
       - main
   workflow_dispatch: { }
+concurrency:
```

Comment:
Please add a blank line before this line for better readability.

Reply:
IMO,a blank line is unnecessarily, so just Add the new line with doc.

Reply:
Added new line